season-of-docs-proposal.md

```diff
@@ -0,0 +1,70 @@
+_This is a project proposal for Google's [Season of Docs](https://developers.google.com/season-of-docs) support._
+
+---
+
+# Improve Exercism's docs to encourage new contributions
+
+## About your organization
+
+Exercism is a not-for-profit organisation that helps people develop and improve their programming skills through practice and mentoring.
+Our platform is made up of over 200 GitHub repositories covering nearly 70 programming languages. 
+Everything is Open Source. Since launching in 2013 we have helped over 500,000 people become better programmers.
+
+Good communication is at the heart of Exercism. 
+The heart of our platform is mentoring, where strangers help each other understand and fill knowledge gaps, and where empathetic communication is essential.
+The product is built by volunteers — so creating a fun, exciting and enticing place to build together is a key part of our mission.
+
+## About your project
+
+### Your project’s problem
+
+One of our challenges is that Exercism is a large project, which can make knowing where to start intimidating and overwhelming.
+Ensuring that contributors have a frictionless contribution experience, are able to find and adhere to our guidance, and (most importantly) feel like they can communicate clearly in the "Exercism way" with our other contributors, is vital.
+
+As we have developed tighter quality-control and implemented firmer standards across different parts of the project, there is more that new contributors have to understand to get started, which has increased the barrier to contribute.
+We need to ensure that new contributors can find both clear introductory paths and comprehensive supporting details when needed.
+Good documentation is absolutely critical to ensuring that new contributors can find pathways to get started, and not get blocked or feel frustration while trying to help.
+We believe it is a crucial part of enabling Exercism to reach its potential.
+
+New contributors not only help Exercism grow, they provide important improvements in the experience for hundreds of thousands of new students who use our platform each year.
+
+As part of the approaching launch of our new version of the site launch soon, we have developed a whole "docs" mini-site. You can preview the WIP version here: https://exercism.lol/docs
```

Comment:
> As part of the approaching launch of our new version of the site launch soon, we have developed a whole "docs" mini-site. You can preview the WIP version here: https://exercism.lol/docs

Suggest removing "soon":  

As part of the approaching launch of our new version of the site, we have developed a whole "docs" mini-site. You can preview the WIP version here: https://exercism.lol/docs

Reply:
https://github.com/exercism/docs/commit/18ee2e723a4225de81b89ade36839ed4b0cdc617 :+1: